subsys/caf/modules/leds.c

```diff
@@ -32,7 +32,15 @@ struct led {
 	struct k_delayed_work work;
 };
 
-#define DT_DRV_COMPAT pwm_leds
+#ifdef CONFIG_CAF_LEDS_PWM
+  #define DT_DRV_COMPAT pwm_leds
+  #define DEFAULT_LABEL(id) "LED_PWM_"#id
+#elif defined(CONFIG_CAF_LEDS_GPIO)
+  #define DT_DRV_COMPAT gpio_leds
+  #define DEFAULT_LABEL(id) "leds"
```

Comment:
why there is only one led? that's against the point.
each group should be a single logical led and each led within a group is a color.

Reply:
There is up to 1 default LED, but there are also LEDs labeled by user.
It's related to Zephyr's implementation. I wanted to let user use LED node defined by default in Zephyr (apart from nodes that are labeled by the user). Detailed explanation in comment above.